cognite/client/data_classes/data_modeling/cdm/v1.py

```diff
@@ -70,7 +70,7 @@ def __init__(
         existing_version: int | None = None,
         type: DirectRelationReference | tuple[str, str] | None = None,
     ) -> None:
-        TypedNodeApply.__init__(self, space, external_id, existing_version, None, type)
+        TypedNodeApply.__init__(self, space, external_id, existing_version, type)
```

Comment:
If we edit this file manually, we cannot regenerate it any more without losing the manual edits. 

Suggest we instead include this changes into the generation and then apply it both to the core and extractor extension models.